src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
```

Comment:
```suggestion
    public String getRandomColor() {
```

Reply:
not fixed

Reply:
not fixed, please go thru PR review history